### PR TITLE
Shanshan's answer

### PR DESCRIPTION
**Pull request by shawnpei**
Thanks for all the help!

### CODE REVIEW

File: answers.sh [hunk 1]
```diff
@@ -1,17 +1,82 @@
...
+    | sort -k7rn \
+    | cut -f4 \
+    | head -1)
+echo "answer-7: $answer_7"
+
+# Question 8: How many unique chromosomes are in genes.hg19.bed.gz?
+answer_8=$(gzcat $hg19bed \
+    | cut -f1 \
+    | sort \
+    | uniq -c \
```
Comment (kriemo) [inline answers.sh]: `-c` is unnecessary

**Comment by jayhesselberth** [comment]
👍

File: answers.sh [hunk 2]
```diff
@@ -1,17 +1,82 @@
...
+answer_2=$(grep -v '^>' $sequence \
+    | wc -l)
+echo "answer-2: $answer_2"
+
+# Question 3: How many unique CpG IDs are in cpg.bed.gz?
+bed="$datasets/bed/cpg.bed.gz"
+answer_3=$(gzcat $bed \
+    | cut -f4 \
+    | sort \
+    | uniq -c \
```
Comment (shawnpei) [inline answers.sh]: Thank you!

File: answers.sh (hunk 1 above)
Comment (shawnpei) [inline answers.sh]: Got it, thank you!

**Pull request closed without merging**